mysite/settings.py

```diff
@@ -66,7 +66,6 @@
 MIDDLEWARE = ['django.middleware.security.SecurityMiddleware',
               'django.contrib.sessions.middleware.SessionMiddleware',
               'django.middleware.common.CommonMiddleware',
-              'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
Why?

Reply:
because when I testing add files, it's blocking.